docs/UserGuide.md

```diff
@@ -47,46 +53,54 @@ VendorVault is a **desktop app for managing supplier contact information and del
+     * Mac users can open the terminal by typing in `Terminal` using the spotlight search bar (accessed by pressing Command + Space). 
      * Linux users can open for the command terminal by searching for the `Terminal` or `Console` in the Applications Menu.
    
-   - 1b. Type in java -version to check if you have Java installed. If you have java installed you should see something like this:
+   - 1b. Type in java -version to check if you have Java installed. If you have Java installed you should see something like this:
      ![Terminal Example showcasing java version](images/intro/terminalJavaVersion.png)
    
-   - 1c. If you do not have Java installed, you can download it from [here](https://www.oracle.com/java/technologies/downloads/#java17?er=221886).
+   - 1c. If you do not have Java installed or are currently on a different version, you can download it from [here](https://www.oracle.com/java/technologies/downloads/#java17?er=221886).
 
 2. Download the latest `vendorvault.jar` file from [here](https://github.com/AY2425S1-CS2103T-W14-1/tp/releases).
    ![Download jar file](images/intro/jarFile.png)
 
 3. Upon downloading the `vendorvault.jar` file, copy the file to the folder you want to use as the _home folder_ for your VendorVault Application.
 
-4. Open the command terminal (same as step 1a), type `cd [path-to-your-folder]` (path to the folder you put the jar file in) and use the `java -jar vendorvault.jar` command to run the application.<br>
-   ![Starting up application](images/intro/startingApp.png)
+4. Open the command terminal (same as step 1a), type `cd [path-to-your-folder]` (path to the folder you placed the jar file in step 3) and use the `java -jar vendorvault.jar` command in the terminal to run the application.<br>
+   ![Starting up application](images/intro/startingApp.png). If you are unsure how to do this step follow these guides below:
+    * Windows users follow this [guide](https://wiki.communitydata.science/Windows_terminal_navigation).
+    * Mac users follow this [guide](https://darkroomphotos.com/mac-terminal-navigate-folders/).
+    * Linux users follow this [guide](https://www.redhat.com/en/blog/Linux-file-navigation-commands).
 
-A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.
 
-<br>
+A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data:
+
 
 ![Ui](images/intro/Ui.png)
+<br>
 
-5. Type commands in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+5. In the application the box that says `Enter command here ... ` is called the command box. This is where you will input in your commands. 
+
+6. The box below the command box is called the `results display box`. This is where the result of your inputted command will be displayed. 
```

Comment:
Would putting the results display box in backticks make the user think that it could be part of a command?